src/exporto0o.ts

```diff
@@ -141,23 +126,27 @@ export async function exportToOo(
     variables.outputFileName = variables.outputFileFullName.substring(0, variables.outputFileFullName.lastIndexOf('.'));
   }
 
+  const env = Object.fromEntries(Object.entries(setting.env ?? {}).map(([n, v]) => [n, renderTemplate(v, variables)]));
+
   // show progress
   progress.setMessage(lang.preparing(outputFileFullName));
   beforeExport && beforeExport();
   progress.show();
 
   const pandocPath = getPlatformValue(globalSetting.pandocPath) ?? 'pandoc';
 
-  const cmdTpl =
+  let cmdTpl =
```

Comment:
These changes are unnecessary. The template render function will do this automatically. Do you understand?

Reply:
The whole changes of this file are also unnecessary.